app/bundles/LeadBundle/EventListener/LeadSubscriber.php

```diff
@@ -256,7 +286,7 @@ public function onNoteDelete(Events\LeadNoteEvent $event)
      */
     public function preLeadMerge(Events\LeadMergeEvent $event)
     {
-        $this->em->getRepository('MauticLeadBundle:LeadEventLog')->updateLead(
+        $this->entityManager->getRepository(LeadEventLog::class)->updateLead(
```

Comment:
Can we inject the repo directly? We don't use the entity manager in this class for anything other than getting repositories.

Reply:
I was just being lazy here as there are 6 repositories in this class. Again, the point of this PR was to get rid of the CommonSubscriber. Everything else is nice to have. I will keep this if you don't mind that much.